test/test_functional_tensor.py

```diff
@@ -1141,5 +1003,148 @@ def test_adjust_gamma(device, dtype, config):
     )
 
 
+@pytest.mark.parametrize('device', cpu_and_gpu())
+@pytest.mark.parametrize('func, args', [(F_t._get_image_size, ()), (F_t.vflip, ()),
```

Comment:
It's no big deal but I find this formatting to be a bit cleaner, and easier to read:
```py
@pytest.mark.parametrize('func, args', [
    (F_t._get_image_size, ()),
    (F_t.vflip, ()),
    (F_t.hflip, ()),
    (F_t.crop, (1, 2, 4, 5)),
    (F_t.adjust_brightness, (0., )),
    (F_t.adjust_contrast, (1., )),
    ...
])
```